python/tests/test_approximate_nearest_neighbors.py

```diff
@@ -444,9 +564,9 @@ def assert_row_equal(r1: Row, r2: Row) -> None:
             "array",
             10000,
             {
-                "nlist": 10,
-                "nprobe": 2,
-                "M": 2,
+                "nlist": 100,
```

Comment:
Do these changes increase test run time significantly?

Reply:
I think it increases a bit. The test case (4 instances) takes 2min30s.